docs/guides/build/cli.md

```diff
@@ -1,4 +1,8 @@
-# CLI Building
+# Build CLI
 
-- Install stable rust toolchain.
-- Go into the [CLI folder](/clients/cli) and `cargo build --release`. To add `lockbook` to your path you can `cargo install --path .`
\ No newline at end of file
+Prerequisites:
+- Stable rust toolchain
+
+Steps:
+- In `/clients/cli` run `cargo build --release`
+- To add `lockbook` to your PATH you can use `cargo install --path .`
```

Comment:
I believe `cargo install --path .` does something _slightly_ different than `build --release`.

@tvanderstad can you `cargo clean`, `cargo build --release` and then `cargo install --path .`?

If the last command fires off a clean build, these aren't the best instructions.

If I'm wrong and it does not fire off a clean build these are fine.

Reply:
yeah, `cargo install` doesn't respect the lockfile without the `--locked` flag; updated this